### PR TITLE
Updated the icons for the buttons, created a new page called "setting…

### PR DESCRIPTION
**Pull request by brooke-k**
…s" to house the settings and linked it to the tab bar. Had to update the typescript to the latest version to allow the app to open, but no other problems.

### CODE REVIEW

**Review by RatikKapoor — CHANGES_REQUESTED** [review]
Can you remove `package-lock.json` and use `yarn install` instead of `rpm install`? Having both `package-lock.json` and `yarn.lock` can get messy and break the build! @brooke-k

**Comment by brooke-k** [comment]
Yep! Not sure how to add on to this pull request, if possible, so I'll close this and submit a new one with the updated changes.

**Review by RatikKapoor — APPROVED** [review]
:shipit:

**Comment by RatikKapoor** [comment]
Whoops, accidentally removed my review. Fixing now

**Review by RatikKapoor — APPROVED** [review]
🚢

File: src/App.tsx [hunk 1]
```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import { Redirect, Route } from 'react-router-dom';
 import { IonApp, IonIcon, IonLabel, IonRouterOutlet, IonTabBar, IonTabButton, IonTabs } from '@ionic/react';
 import { IonReactRouter } from '@ionic/react-router';
-import { ellipse, square, triangle } from 'ionicons/icons';
+import { carOutline, musicalNoteOutline, navigateOutline, menuOutline } from 'ionicons/icons';
```
Comment (b4yc) [inline src/App.tsx:5]: hehe awesome

**Review by b4yc — COMMENTED** [review]
:shipit:

**Review by b4yc — APPROVED** [review]

**Pull request merged**